examples/qemist_cloud_hardware_experiments_braket.ipynb

```diff
@@ -111,7 +111,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from tangelo.linq.qpu_connection import job_submit, job_status, job_cancel, job_result, job_estimate"
+    "from tangelo.linq.qpu_connection import QEMISTCloudConnection\n",
+    "qemist = QEMISTCloudConnection()"
```

Comment:
Let's not call a class a name that has been used for our product.
`qcloud_connection` or `qemist_connection` may be better.

The term "connection" is rather well defined: https://docs.microsoft.com/en-us/sql/ado/reference/ado-api/connection-object-ado?view=sql-server-ver16